go.mod

```diff
@@ -251,3 +250,6 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.17.1 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+// remove this when https://github.com/mdlayher/ndp/pull/32 gets merged
+replace github.com/mdlayher/ndp => github.com/xliuxu/ndp v0.0.0-20240926134643-8cf547505092
```

Comment:
should it be updated?

Reply:
i thought it was updated. but i pushed the latest changes to the wrong upstream.. I will update and address the latest comments.